web/sensors/v0.7/changelog.md

```diff
@@ -0,0 +1,196 @@
+
+  - Support [flatten](./pvql#flatten) function for `url` type.
+  - Added new VLAN related fields. It is possible now to use the
+    operator `IN` with a list of VLANs, For example, `traffic FROM tcp
+    WHERE 5 IN server.vlans`.
+      - Added new field
+        [client.vlans](./definitions#field-client.vlans).
+      - Added new field
+        [server.vlans](./definitions#field-server.vlans).
+      - Added new field
+        [source.vlans](./definitions#field-source.vlans).
+      - Added new field [dest.vlans](./definitions#field-dest.vlans).
+      - Added new field
+        [client.vlans.outer](./definitions#field-client.vlans.outer).
+      - Added new field
+        [server.vlans.outer](./definitions#field-server.vlans.outer).
+      - Added new field
+        [source.vlans.outer](./definitions#field-source.vlans.outer).
+      - Added new field
+        [dest.vlans.outer](./definitions#field-dest.vlans.outer).
+      - Added new field
+        [client.vlans.inner](./definitions#field-client.vlans.inner).
+      - Added new field
+        [server.vlans.inner](./definitions#field-server.vlans.inner).
+      - Added new field
+        [source.vlans.inner](./definitions#field-source.vlans.inner).
+      - Added new field
+        [dest.vlans.inner](./definitions#field-dest.vlans.inner).
+      - Added new field
+        [client.vlans.count](./definitions#field-client.vlans.count).
+      - Added new field
+        [server.vlans.count](./definitions#field-server.vlans.count).
+      - Added new field
+        [source.vlans.count](./definitions#field-source.vlans.count).
+      - Added new field
+        [dest.vlans.count](./definitions#field-dest.vlans.count).
+  - Added new fields for time window exclusion:
+      - Added new field
+        [time\_exclusion.business\_hours](./definitions#field-time_exclusion.business_hours).
+      - Added new field
+        [time\_exclusion.maintenance\_windows](./definitions#field-time_exclusion.maintenance_windows).
+      - Added new field
+        [time\_exclusion.any](./definitions#field-time_exclusion.any).
+  - Added new fields for DNS-issued hostnames:
+      - Added new field
+        [client.hostname](./definitions#field-client.hostname).
+      - Added new field
+        [server.hostname](./definitions#field-server.hostname).
+      - Added new field
+        [source.hostname](./definitions#field-source.hostname).
+      - Added new field
+        [dest.hostname](./definitions#field-dest.hostname).
+      - Added new field
+        [netflow.hostname](./definitions#field-netflow.hostname).
+
+### Deprecated
+
+  - Single VLAN fields are deprecated by fields containing lists of
+    VLANs.
+      - Deprecated field [client.vlan](./definitions#field-client.vlan).
+      - Deprecated field [server.vlan](./definitions#field-server.vlan).
+      - Deprecated field [source.vlan](./definitions#field-source.vlan).
+      - Deprecated field [dest.vlan](./definitions#field-dest.vlan).
+
+## \[0.6\]
```

Comment:
So, we get a changelog for each version, each including the complete changelog of previous versions (hence, with a lot of repetitions)?

This is a bit weird IMHO, to be discussed :slightly_smiling_face: